doc/tutorial/fcm_specifics

```diff
@@ -0,0 +1,1484 @@
+--------------------------------------------------------------------------------
+Merge: /specifics/branches/dev/$LOGNAME/branch_conflict_tree@46
+ c.f.: /specifics/trunk@643
+-------------------------------------------------------------------------dry-run
+--- Merging r44 through r46 into '.':
+   C src/utility
+   A src/utility/tool_list.txt
+D    tree_conflict.txt
+Summary of conflicts:
+  Tree conflicts: 1
+-------------------------------------------------------------------------dry-run
+Would you like to go ahead with the merge?
+Enter "y" or "n" (or just press &lt;return&gt; for "n"): <font color="black">y</font>
+Merge succeeded.
+--------------------------------------------------------------------------actual
+--- Recording mergeinfo for merge of r42 through r43 into '.':
+ U   .
+--- Merging r44 through r46 into '.':
+   C src/utility
+   A src/utility/tool_list.txt
+D    tree_conflict.txt
+--- Recording mergeinfo for merge of r44 through r46 into '.':
+ U   src/utility/tool_list.txt
+ G   .
+Summary of conflicts:
+  Tree conflicts: 1
+--------------------------------------------------------------------------actual
+</font>
+fcm st
+<font color="blue">
+ M      .
+      C src/utility
+      >   local dir obstruction, incoming dir add upon merge
+ M      src/utility/tool_list.txt
+D       tree_conflict.txt
+Summary of conflicts:
+  Tree conflicts: 1
+</font>
+fcm resolve --accept working src/utility
+<font color="blue">
+Resolved conflicted state of 'src/utility'
+</font>
+</pre>
+                <br/>
+                The conflict has been resolved.
+                <br/>
+                <br/>
+                We will use this command further in the <a href="#S5">tree conflicts</a> 
+                sub-section.
+            </div>
+            <div class="well">
+                <h4>Tree conflicts</h4>
+                Tree conflict, which are otherwise known as a structural or filesystem-based 
+                conflicts.
+                <br/>
+                These can be more complicated that text conflicts.
+                <br/>
+                <code>fcm st</code> returns the following when there is a tree conflict on the
+                repository, shown by a "C" present in column 7 of the returned output:
+                <pre>       C src/utility/my_tools</pre>
+                <br/>
+                <code>fcm conflicts</code> should manually resolve the problem by prompting us 
+                to choose a course of action.
+                We can either keep the file as it was before the merge (keeping the local copy),
+                or accept the external changes to the file.
+                <br/>
+                <br/>
+                FCM has issues when the same directory has been added to 2 branches.
+                The problem can occur when merging two branches that both have the same
+                directory added.
+                I.e. JoeBloggs added src/utility/my_tool to a branch and we also added the
```

Comment:
e.g. rather than i.e.

Am slightly confused as we were talking about directories rather than files.